app/src/main/res/values/strings.xml

```diff
@@ -235,6 +235,19 @@
   <string name="TestResults_Details_Circumvention_Tor_Table_Header_Type">Type</string>
   <string name="TestResults_Details_Circumvention_Tor_Table_Header_Connect">Connect</string>
   <string name="TestResults_Details_Circumvention_Tor_Table_Header_Handshake">Handshake</string>
+  <string name="TestResults_Details_Circumvention_Riseupvpn_Blocked_Hero_Title">Likely blocked</string>
+  <string name="TestResults_Details_Circumvention_Riseupvpn_Reachable_Hero_Title">Working</string>
+  <string name="TestResults_Details_Circumvention_Riseupvpn_Blocked_Content_Paragraph">[RiseupVPN](https://riseup.net/vpn) appears to be blocked.</string>
+  <string name="TestResults_Details_Circumvention_Riseupvpn_Reachable_Content_Paragraph">We were able to successfully connect to RiseupVPN\'s bootstrap server and VPN gateways. This means that [RiseupVPN](https://riseup.net/vpn) should work.</string>
+  <string name="TestResults_Details_Circumvention_Riseupvpn_Table_Header_Api">Bootstrap server</string>
+  <string name="TestResults_Details_Circumvention_Riseupvpn_Table_Header_Openvpn">OpenVPN connections</string>
+  <string name="TestResults_Details_Circumvention_Riseupvpn_Table_Header_Bridge">Bridged connections</string>
+  <string name="TestResults_Overview_Circumvention_Riseupvpn_Api_Blocked">blocked</string>
+  <plurals name="TestResults_Overview_Circumvention_Riseupvpn_Blocked">
+    <item quantity="one">%d blocked</item>
```

Comment:
NOTE TO SELF: 
remove these quantity and add
- TestResults_Overview_Circumvention_Riseupvpn_Blocked_Singular
- TestResults_Overview_Circumvention_Riseupvpn_Blocked_Plural

Reply:
@lorenzoPrimi I think it would be better to generally use quantity strings, because there are languages that have more than 1 plural form. The translation would be grammatically wrong in these cases.

Reply:
@lorenzoPrimi this snippet illustrates it nicely with the Russian example of pineapples.
https://gist.github.com/Eeyore741/a0228dd9b7551daaf1fe8fc11a3548fc#file-plurals_en_ru-csv

Reply:
Unfortunately this would bring our translation system based on transifex, it's a much bigger change that can't be handled in this PR.